scripts/test.sh

```diff
@@ -53,6 +65,20 @@ else
   echo "***** Print conda info *****"
   conda info
   conda list
+  conda clean -yi
```

Comment:
```suggestion
  conda clean --yes --index-cache
```
Just kicking off new builds.